### ISSUE TITLE
Getting results from Lora

### ISSUE THREAD

**Issue by arose26** [davidbrowne17/chatterbox-streaming#9]
Edit: just an update on this. I realized that it was loading the pip installed vanilla Chatterbox and hence why it looked for .safetensors. Fixing the paths, I confirmed it could load .pt LORA model. I also confirmed it was loading the model I trained as renaming a file makes it error. Nevertheless, the output result sounds just like default model. What am I doing wrong?

------

I ran Lora with default settings and it went to the end of 10 epochs. My audio inputs combined lasted around 10 mins. 

Last message was

> Epoch 10 - Train Loss: 1.2860, Val Loss: 1.2088

The output folder looked like this

`checkpoint_epoch0_step5.pt   checkpoint_epoch3_step20.pt  checkpoint_epoch6_step35.pt  checkpoint_epoch9_step50.pt
checkpoint_epoch1_step10.pt  checkpoint_epoch4_step25.pt  checkpoint_epoch7_step40.pt  final_lora_adapter.pt
checkpoint_epoch2_step15.pt  checkpoint_epoch5_step30.pt  checkpoint_epoch8_step45.pt  merged_model`

and merged_model is what I kept which looked like this

`conds.pt  s3gen.pt  t3_cfg.pt  tokenizer.json  ve.pt`

**Comment by arose26** [comment]
Another update, I lowered the learning rate and ran it for twice the epochs. It still winds up no different to the default model.

**Comment by mjsb212** [comment]
I sucessfully finetuned lora with this repo, however only for a male voice & everything will everything be baked in to the T3 part but for Male only, I sucessfully finetuned a new voice in English and also with a "Non English accent. With a female voice I need to train and run s3gen with T3 & with the trained voice reference on inference and I get great results. I believe these oberservations are because the original model is default to a male voice and no idea if this can be changed unless completly training from scratch which I dont think is currently possible.

**Comment by mjsb212** [comment]
I sucessfully finetuned lora with this repo, however only for a male voice & everything can be baked in to the T3 part sucessfully -- but for Male only. I sucessfully finetuned a new Male voice in English and also with a Male "Non English accent". With a female voice I need to train and run the s3gen with T3 & with the trained voice reference set on inference and I get great results and a Female voice. I believe these oberservations are because the original model is default to a male voice, and no idea if this can be changed unless completly training from scratch, which I dont think is currently possible.

**Comment by mjsb212** [comment]
I sucessfully finetuned lora with this repo, however only for a male voice & everything can be baked in to the T3 part sucessfully -- but for Male only. I sucessfully finetuned a new Male voice in English and also with a Male "Non English accent". With a female voice if I train I get results that are exactly the same a the original model, so no new voice is observed. BUT if train and run the s3gen with T3 & with the trained voice as a eference using S3gen set on inference--I get great results and a great Female voice like I trained. I believe these oberservations are because the original model is default to a male voice, and no idea if this can be changed unless completly training from scratch, which I dont think is currently possible.

**Comment by mjsb212** [comment]
I sucessfully finetuned lora with this repo, however only for a male voice & everything can be baked in to the T3 part sucessfully -- but for Male only. I sucessfully finetuned a new Male voice in English and also with a Male "Non English accent". With a female voice if I train I get results that are exactly the same as the original model, so no new voice is observed. BUT if train all models and run the s3gen with T3 & with the trained voice as a reference using S3gen set on inference--I get great results and a great Female voice like I trained. I believe these oberservations are because the original model is default to a male voice, and no idea if this can be changed unless completly training from scratch, which I dont think is currently possible.

**Comment by mjsb212** [comment]
I sucessfully finetuned lora with new voices using this repo. T3 model trains the speaker style, accent & rhythm / cadence. Then s3gen model trains the voice pitch and tambre. You must then set a reference voice on inference to a sample of the voice you just trained and T3 and S3gen will run and give you the new voice & style. Results are very good so far.

**Comment by ChrisJohnNewton** [comment]
> I sucessfully finetuned lora with new voices using this repo. T3 model trains the speaker style, accent & rhythm / cadence. Then s3gen model trains the voice pitch and tambre. You must then set a reference voice on inference to a sample of the voice you just trained and T3 and S3gen will run and give you the new voice & style. Results are very good so far.

It seems weird that we'd need to set a reference voice on inherence after just training the voice model. Are you sure you're not just seeing one-shot cloning here and that your LORA-trained merged model is actually being used? I am experiencing the same issue as the original poster. Thanks for any info you may have.
